app/uk/gov/hmrc/gatekeeperemail/services/EmailService.scala

```diff
@@ -55,13 +57,19 @@ class EmailService @Inject()(emailConnector: GatekeeperEmailConnector,
   def sendEmail(emailUUID: String): Future[Email] = {
     for {
       email <- emailRepository.getEmailData(emailUUID)
-      emailRequestedData = SendEmailRequest(email.recipients, email.templateData.templateId, email.templateData.parameters,
-        email.templateData.force, email.templateData.auditData, email.templateData.eventUrl)
-      _ <- emailConnector.sendEmail(emailRequestedData)
+      _ <- persistInEmailQueue(email)
       _ <- emailRepository.updateEmailSentStatus(emailUUID)
     } yield email
   }
 
+  private def persistInEmailQueue(email: Email):  Future[Email] = {
+    email.recipients.map(elem =>
+      sentEmailRepository.persist(SentEmail(LocalDateTime.now(), LocalDateTime.now(), UUID.fromString(email.emailUUID),
```

Comment:
As per other comment, may be beneficial to map to a collection of SentEmails and then save that in one repository call.